projects/core/services/breakpoint.service.ts

```diff
@@ -0,0 +1,44 @@
+import {Inject, Injectable} from '@angular/core';
+import {WINDOW} from '@ng-web-apis/common';
+import {TuiMedia} from '@taiga-ui/core/interfaces';
+import {TUI_MEDIA} from '@taiga-ui/core/tokens';
+import {fromEvent, merge, Observable} from 'rxjs';
+import {map, share, startWith} from 'rxjs/operators';
+
+/**
+ * Service to provide the current breakpoint based on Taiga UI's media queries
+ */
+@Injectable()
+export class TuiBreakpointService extends Observable<string | null> {
```

Comment:
maybe `Observable<'mobile' | 'desktopSmall' | 'desktopLarge' | null>` instead of `Observable<string | null>` ?